hello-signing.md

```diff
@@ -0,0 +1,145 @@
+Since they've never heard of Wabbit Networks, they're a little reluctant to run network software without some validations.
+ACME Rockets has a policy to only import Docker Hub certified content, and other approved vendors.
+
+Wabbit Networks works with Docker Hub to get certified, to help with their customer confidence.
+
+ACME Rockets will only deploy software that's been scanned and approved by the ACME Rockets security team. They know it's been approved because all approved software has been signed by the ACME Rockets security team.
+
+## Getting Started
+- Setup a few environment variables.  
+  >Note see [Simulating a Registry DNS Name](#simulating-a-registry-dns-name) to use `registry.wabbit-networks.io`
+  ```bash
+  export PORT=5000
+  export REGISTRY=localhost:${PORT}
+  export REPO=${REGISTRY}/net-monitor
+  export IMAGE=${REPO}:v1
+  ```
+- Install [Docker Desktop](https://www.docker.com/products/docker-desktop) for local docker operations
+- Run a local instance of the [CNCF Distribution Registry][cncf-distribution]
+  ```bash
+  docker run -d -p ${PORT}:5000 ghcr.io/oras-project/registry:latest
+  ```
+- Acquire the Notation CLI  
+Notation releases can be found at: [Notation Releases][notation-releases]  
+  ```bash
+  #LINUX, including WSL
+  curl -Lo notation.tar.gz https://github.com/shizhMSFT/notation/releases/download/v0.5.2/notation_0.5.2_linux_amd64.tar.gz
+  tar xvzf notation.tar.gz -C ~/bin notation
+  ```
+
+## Building and Pushing
+- Build and Push the `net-monitor` software
+  ```bash
+  docker build -t $IMAGE https://github.com/wabbit-networks/net-monitor.git#main
+
+  docker push $IMAGE
+  ```
+- List the image, and any associated signatures
+  ```bash
+  notation list $IMAGE
+  ```
+  At this point, the results are empty, as there are no existing signatures
+
+## Signing a Container Image
+
+To get things started quickly, the Notation cli supports generating self signed certificates. As you automate the signing of content, you will most likely want to create and store the private keys in a key vault. (Detailed production steps will be covered later)
+
+- Generate a self-signed test certificate for signing artifacts
+  The following will generate a self-signed X.509 certificate under the `~/config/notation/` directory
+  ```bash
+  notation cert generate-test --default "wabbit-networks.io"
+  ```
+- Sign the container image
+  ```bash
+  notation sign $IMAGE
+  ```
+- List the image, and any associated signatures
+  ```bash
+  notation list $IMAGE 
+  ```
+
+## Verify a Container Image Using Notation Signatures
+
+To avoid a Trojan Horse attack, and before pulling an artifact into an environment, it is important to verify the integrity of the artifact based on the identity of an entity you trust. Notation uses a set of configured public keys to verify the content. The `notation cert generate-test` command created the public key, however it must be implicitly added for verification to succeed.
```

Comment:
```suggestion
To avoid a Trojan Horse attack, and before pulling an artifact into an environment, it is important to verify that the artifact was unmodified after it was created (integrity), and from an trusted entity (authenticity). Notation uses a set of configured public keys that represent trusted entities, to verify the content. The `notation cert generate-test` command created the public key, however it must be implicitly added for verification to succeed.
```